apps/web/components/dashboard/settings/UserDetails.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
```

Comment:
This sounds like it can be a server component instead and you can use the `users.whoami` trpc endpoint.

Reply:
Thanks, changed it to the server component. I'm new to next 😅